packages/design-system/src/components/FilterTag/FilterTag.tsx

```diff
@@ -0,0 +1,104 @@
+import { ClearIcon } from '../ClearIcon';
+import React from 'react';
+import classNames from 'classnames';
+
+const actionableKeys = ['Enter', 'Backspace', 'Delete'];
+const idFromValue = (value: string) => `${value}-tag`;
+
+export interface FilterTagProps {
+  /**
+   * Id for filter tag button. If not provided, a unique id will be automatically generated and used.
+   */
+  id?: string;
+  /**
+   * Class to be applied to the outer `<div>` that contains filter tag button.
+   */
+  className?: string;
+  /**
+   * Text for the filter tag
+   */
+  label: string;
+  /**  
+   *
+   Value for filter tag 
+   */
+  value: string;
+  /**
+   *  For screenreaders, text to read for removal
+   */
+  removeText: string;
+  /**
+   * When false, filter tag is not removable
+   */
+  readOnly?: boolean;
+  /**
+   * Function to call when filter tag is dismissed, will return value
+   */
+  onClear?: (value: string) => void;
+}
+
+interface FilterTagState {
+  tagDismissed: boolean;
```

Comment:
What kind of use case would this have? The only way we use it so far in Enroll apps is where the this component's parent is in charge of managing this kind of state. I foresee this self-managed visibility resulting in a bunch of zombie filter tag instances.

Reply:
I think you're right. I'm going to remove & just change the example